### PR TITLE
React Migration

### PR DESCRIPTION
**Pull request by mcao**
To Do:
- Implement new interface into the website (see design mockups on Figma)
- Figure out how to dynamically insert data from database on donation links
- Make suggestion form a native React/JS form that pops up instead of diff. page
- Check Trello cards for other to-dos

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <meta name="theme-color" content="#f2994a" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <meta name="og:image" content="https://donatecovid19.org/img/embed.png" />
+  <!-- ^^ kind of a weird fix but it works for now -->
+  <meta name="og:title" content="Donate to Fight COVID-19" />
+
+  <meta name="twitter:url" content="https://donatecovid19.org">
+  <meta name="twitter:title" content="Donate to Fight COVID-19">
+  <meta name="twitter:description" content="Find a donation fund worldwide to support coronavirus relief efforts.">
```
Comment (ykatieli) [inline public/index.html:19]: Nit: twitter?

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        by{" "}
+        <a href="#" rel="noopener noreferrer" target="_blank">
+          Allison
+        </a>{" "}
+        <a
+          href="https://www.youtube.com/channel/UC9xFO-llZ2ontatfj9LtLxw"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          Jack
```
Comment (ProjectsByJackHe) [inline src/components/Footer.jsx:20]: could we change this to Jack He?

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  return (
+    <div className="footer">
+      <footer>
+        Made with{" "}
+        <span role="img" aria-label="heart">
+          &#129505;
+        </span>{" "}
+        by{" "}
+        <a href="#" rel="noopener noreferrer" target="_blank">
+          Allison
```
Comment (ykatieli) [inline src/components/Footer.jsx:13]: Nit: All names should be uniform - first and last name

File: src/components/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function Home() {
+  return (
+    <div className="form">
+      <title>Suggest a Donation Link!</title>
+      <iframe
+        title="Donation Link Google Form"
+        src="https://docs.google.com/forms/d/e/1FAIpQLSdvsS4TEJiDKwphNyaMeoBa4d8XbjsIvQd2hjucI1Ev5p5q5w/viewform?embedded=true"
```
Comment (ykatieli) [inline src/components/Form.jsx:9]: Nit: delete extra line. Is the code linted?

File: public/index.html (hunk 1 above)
Comment (mcao) [reply public/index.html:19]: Just to pretty up the Twitter embed - Twitter does embeds diff. than most platforms

File: src/components/Form.jsx (hunk 4 above)
Comment (mcao) [reply src/components/Form.jsx:9]: Linter I'm using actually enforces this styling

**Pull request merged**